### PR TITLE
Form 10-10EZR keyboard only test

### PR DESCRIPTION
**Pull request by dellerbie**
Keyboard only Cypress test for Form 10-10EZR

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [ ] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Related issue(s)

- department-of-veterans-affairs/va.gov-team#80252

## Testing done

Added a new Cypress E2E test for this

## What areas of the site does it impact?

Form 10-10EZR

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: src/applications/ezr/tests/e2e/fixtures/data/maximal-test.json [hunk 1]
```diff
@@ -152,8 +152,12 @@
       "suffix": "Jr."
     },
     "view:gulfWarServiceDates": {
-      "gulfWarStartDate": "1990-09-XX",
```
[the remark this answers is not in the document]
Comment (longmd) [reply src/applications/ezr/tests/e2e/fixtures/data/maximal-test.json:155]: The `-XX` is how the `va-memorable-date` formats the values when the format is month/day only. Id suggest reverting this to provide accurate data formatting for the component.

**Review by longmd — CHANGES_REQUESTED** [review]
Please revert the date formatting in the maximal test to accurately reflect how that gets formatted.

File: src/applications/ezr/tests/e2e/fixtures/data/maximal-test.json [hunk 2]
```diff
@@ -155,6 +155,10 @@
       "gulfWarStartDate": "1990-09-XX",
       "gulfWarEndDate": "1991-01-XX"
     },
+    "view:toxicExposureDates": {
```
Comment (longmd) [inline src/applications/ezr/tests/e2e/fixtures/data/maximal-test.json:158]: This data should match the same month/year format as the Gulf War dates. Let's replace the "day" values with `XX`.

**Review by longmd — CHANGES_REQUESTED** [review]

File: src/applications/ezr/tests/e2e/fixtures/data/maximal-test.json [hunk 3]
```diff
@@ -155,6 +155,10 @@
...
+      "toxicExposureStartDate": "1990-09-XX",
+      "toxicExposureEndDate": "1990-09-XX"
```
Comment (longmd) [inline src/applications/ezr/tests/e2e/fixtures/data/maximal-test.json:160]: Good extra test of the same-as date selection! 👍🏼

**Review by longmd — APPROVED** [review]

**Review by mkerns1 — APPROVED** [review]
LGTM!

**Pull request merged**